src/components/EditSections/EditUserInfo/EditUserInfo.js

```diff
@@ -88,6 +90,7 @@ class EditUserInfo extends React.Component {
               name="personal.lastName"
               id="adduser_lastname"
               component={TextField}
+              props={autoFocusThisField}
```

Comment:
is it better in something than just `autoFocus`?

Reply:
less vague

Reply:
Sorry, I'm with @aliaksei-chumakou on this one again.

Reply:
maybe "vague" - is too subjective thing, `props` is a reserved React word, so to me it's not obvious why we pass `id`, `required`, `fullWidth` separately, but `autoFocus` in other way. It's not an error to do like this, but maybe a question of project code style and consistency

Reply:
About the prop `pops`, exactly!
But it's an API of [redux-form Field](https://redux-form.com/6.4.3/docs/api/field.md/)

And for me, add upon this this
```js
const autoFocus = { autoFocus: true };
```
would be too much for my taste ))

Can we maybe agree on `AUTO_FOCUS` or `AUTO_FOCUS_PROPS`, in all-caps, like the constants usually done?

Reply:
@axelhunn  right, but it will not work with react-final-form, IIUC https://final-form.org/docs/react-final-form/types/FieldProps

Reply:
Should I make a universal thing like
```js
const TexFieldAutofocused = props => <TextField {...props} autoFocus />;
//...
<Field
  //...
  name="personal.lastName"
  component={TexFieldAutofocused}
/>
```

In final-form this would work just like this, in more elegant way
```js
<Field
  //...
  component={TextField}
  autoFocus
/>
```

Reply:
I think 
```
<Field
  //...
  component={TextField}
  autoFocus
/>
```
works with both frameworks, no?

Reply:
Seems like it should work.
My mistake, have read the docs poorly.

Reply:
fixed